src/fs-shim.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
 const fs = (typeof window === 'object' || typeof self === 'object') ? null
   : eval('require("fs")')
```

Comment:
`eval` considered harmful, except that this eval has no user input associated with it

Reply:
Just for the sake of my curiosity, what does the eval even do here?

Reply:
in the event of it not being in the browser (no `window` object), or a web worker (no `self` object) it simply calls the result of evaluating `require("fs")`, which is... `require('fs')`

It's a "clever" runtime check for fs and it allows downstream projects to not have to deal with `fs` in their own webpack configs

Reply:
Makes sense. I didn't catch on how it prevents an error in the browser with the undefined `require` function